public/app/features/dashboard-scene/scene/layout-default/DefaultGridLayoutManager.tsx

```diff
@@ -60,6 +61,14 @@ export class DefaultGridLayoutManager
 
   public readonly descriptor = DefaultGridLayoutManager.descriptor;
 
+  constructor(state: DefaultGridLayoutManagerState) {
+    super(state);
+
+    this.state.grid.setState({
+      onDragStart: (_evt, panel) => getLayoutOrchestratorFor(panel)?.startDraggingSync(panel),
+    });
```

Comment:
```suggestion
  constructor(state: DefaultGridLayoutManagerState) {
    super({
       ...state, 
       onDragStart: (_evt, panel) => getLayoutOrchestratorFor(panel)?.startDraggingSync(panel)
    });
  }
```

@bfmatei but why the need for setState? why not ^ ?

Reply:
^^ the `onDragStart` is on the `SceneGridLayout` (the `grid` prop), not on the layout (`DefaultGridLayoutManager`)

Reply:
I see, wonder if making this a state property then was wrong

Reply:
I didn't merge the Scenes PR. An alternative would be to emit an event and capture it in the layout manager. That would allow us avoid the state prop and keep the implementation decoupled from the layout manager.

Reply:
https://github.com/grafana/scenes/pull/1059/files#r1965602183

Reply:
Updated!